test/configCases/web/prefetch-preload-module-jsonp/webpack.config.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
 		module: true,
 		filename: "bundle0.mjs",
 		chunkFilename: "[name].js",
-		crossOriginLoading: "anonymous"
+		crossOriginLoading: "anonymous",
+		chunkFormat: "array-push"
```

Comment:
I'm a little afraid of this.

Before - we use `array-push` by default, if no `browserslist` or `output.environment.dynamicImport: true`
Now - we use `module`, because by default `output.environment.dynamicImport` is `conditionallyOptimistic` when no `browserslist` or `output.environment.dynamicImport` is undefined:
```js
F(environment, "dynamicImport", () =>
		conditionallyOptimistic(
			/** @type {boolean | undefined} */ (tp && tp.dynamicImport),
			output.module
		)
	);
```

But because `outputModule` is in `experiments` I think it is good idea and better DX